lib/dry/logic/predicates.rb

```diff
@@ -254,8 +254,13 @@ def uri_rfc3986?(input)
           format?(URI::RFC3986_Parser::RFC3986_URI, input)
         end
 
-        def respond_to?(method, input)
-          input.respond_to?(method)
+        def respond_to?(*args)
```

Comment:
Probably worth a comment above this explaining how we're delegating back to Object's `respond_to?` for compatibility reasons?